ms/second-review-responses.tex

```diff
@@ -166,23 +196,65 @@
 
-    Additionally, the neighborhood size as defined by the authors (see \llname{ll:nw_defn}) is a function of the census density rather than the effective density of individuals. I am therefore not fully convinced that the range of values for this parameter is comparable to that given in Table 1.
+Additionally, the neighborhood size as defined by the authors (see \llname{ll:nw_defn}) is a function of the census density rather than the effective density of individuals. I am therefore not fully convinced that the range of values for this parameter is comparable to that given in Table 1.
 \end{point}
 
 \reply{
+    You are entirely correct here; apologies for the confusion.
+    To clarify the definition of $\sigma$ and discuss these issues somewhat,
+    we have added a new paragraph at \llname{ll:sigma_effective}.
+    As we discuss there, we still calculate $N_W = 4 \sigma^2 \rho$,
+    where $\sigma$ is the mean mother-child dispersal distance
+    (not including the distance between mates)
+    and $\rho$ is the census population density.
+    This is not the quantity that is predicted
+    to best describe geographic patterns of genetic variation,
+    which would use \emph{effective} versions of both $\sigma$ and $\rho$.
+    This does not bother us for a few reasons.
+    First, we do not attempt to match statistics in our simulations
+    to theoretical results (e.g., the Wright--Malec\'ot formula).
+    Instead, we use $N_W$ as a proxy for ``strength of spatial structure'',
+    and in these simulations, the ``effective'' $N_W$ certainly has a monotonic
+    relationship with the $N_W$ we compute.
+    The remaining question is whether, as the reviewer asks,
+    the actual values of $N_W$ are comparable to those computed in the wild.
+    This depends, of course, on how those values were estimated in wild populations.
+    In our opinion, a straightforward estimate of $N_W$
+    using observed dispersal distances and census density or reproductive adults
+    (similar to how we compute $N_W$ for simulations)
+    is likely more robust and reflective of modern demography
+    than most methods to estimate ``effective'' vesions.
+    
+    Furthermore, our imprecisions roughly cancel out.
+    As now noted in the text, an average of one and two dispersal distances
+    is a better model for the mean parent-offspring distance;
+    this would give us a value of $\sqrt{3/2} \sigma$,
+    so by using only $\sigma$ we have lowered $N_W$ by a factor of about 2/3.
+    However, effective population density is lower than census density
+    by a factor of also around 2/3,
+    which would cancel the first.
+    So, we are fairly confident that our values of $N_W$ match the theoretical versions
+    to within the precision we require (which is only order-of-magnitue, really).
+
```

Comment:
I had a go at rewriting what you had, essentially; but have left both in here. Really, we should probably just say "yes, you're right - see the additional explanation". Since they bring up the "range of values", I guess that this (very nice) observation about factors of 3/2 cancelling is OK to put in the response but not the paper, but we should aim for it to be as short as possible, really. Want to edit this stuff together and I'll have another look?

Reply:
I'll take another look at this today